lessons/guesser.rb

```diff
@@ -0,0 +1,45 @@
+puts "Think of a number from 1 to 100."
+puts "I try to guess your number."
+puts "Answer only 'too low', 'too high' or 'correct.'"
+number_guess = 0
+lowest = 1
+highest = 100
+
+loop do
+    number_guess += 1
+    guess = (lowest + highest) / 2
+    puts "Is your number #{guess}?"
+    answer = gets.chomp.downcase
+    if answer == "too low"
+        if guess >= highest
```

Comment:
This is good, but I think you can get the same result by checking equality, since the `guess` should never be higher than the `highest`, right?